apps/builder/app/builder/features/ai/hooks/media-recorder.ts

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable */
```

Comment:
Why suppress everything?

Reply:
Forget to delete, during development eslint have too many rules I don't want to think about